### ISSUE TITLE
Momentum Contrast for Unsupervised Visual Representation Learning 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#6]
# TL;DR
Unsupervised representation learning, MoCo in which performs like metric learning with a dictionary. the key network gradually  updating to be closer to query network. Not only is it necessary to perform linear regression with ImageNet's latent representation, but it can also improve the accuracy of object detection when used in advance learning.
![image](https://user-images.githubusercontent.com/24524018/89173151-c5f6a680-d5be-11ea-9200-c2dbc61289ef.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1911.05722 

# Submission Dates(yyyy/mm/dd)
2019/11/13

# Authors and institutions
Kaiming He, Haoqi Fan, Yuxin Wu, Saining Xie, Ross Girshick
- Facebook AI Research
# Methods

# Results

# Comments
[code](https://github.com/facebookresearch/moco)
